src/util/colorScale.ts

```diff
@@ -183,13 +183,13 @@ function createScaleForGenotype(
   const range = [unknownColor, ...genotypeColors.slice(0, trueValues.length)];
   // Bases are returned by orderOfGenotypeAppearance in order, unknowns at end
   if (legendValues.indexOf("-") !== -1) {
-    range.push(rgb(217, 217, 217));
+    range.push(rgb(217, 217, 217).formatHex());
```

Comment:
FYI, [here's an example URL](https://next.nextstrain.org/mpox/clade-IIb?c=gt-OPG001_1&gt=OPG001.1-) and the results are unchanged with this PR:

<img width="196" alt="image" src="https://github.com/user-attachments/assets/6ad7fdf7-f457-4ca4-8f79-5ad96317d123">